Patches/PlayerControlPatch.cs

```diff
@@ -66,19 +88,16 @@ public static bool Prefix(PlayerControl __instance, [HarmonyArgument(0)] PlayerC
             {
                 Spiritcaller.HauntPlayer(target);
             }
-
-            __instance.RpcResetAbilityCooldown();
-            return true;
+            angel.RpcResetAbilityCooldown();
+            return false;
         }
 
-        if (__instance.Is(CustomRoles.Sheriff))
+        if (angel.Is(CustomRoles.Sheriff) && angel.Data.IsDead)
         {
-            if (__instance.Data.IsDead)
-            {
                 Logger.Info("Blocked protection", "CheckProtect");
-                return false;
-            }
+                return false; // What is this for? sheriff dosen't become guardian angel lmao
         }
+        
         return true;
```

Comment:
I think this code was needed before CrewmateGhost and ImpostorGhost appeared, so the sheriff was automatically assigned a Guardian Angel

Not sure about this, but I think so